test/performance/perf-kinesis/mzcompose.yml

```diff
@@ -7,15 +7,6 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
-
```

Comment:
Is this not supported by CI?

Reply:
possibly these clash with other tests running on the same host?